modules/ocf_printhost/files/monitor-cups

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+"""
+Collects metrics from CUPS and writes them to a text file in the Prometheus
+metrics format.
+"""
+
+import sys
+
+import cups
+from prometheus_client import CollectorRegistry
+from prometheus_client import Gauge
+from prometheus_client import write_to_textfile
+
+
+def main():
+    registry = CollectorRegistry()
+
+    classes = Gauge(
+        'cups_class',
+        'Existence of printer on CUPS class',
+        ['class', 'printer'],
+        registry=registry,
+    )
+
+    conn = cups.Connection()
+    for cups_class, printers in conn.getClasses().items():
+        for printer in printers:
+            classes.labels(**{'class': cups_class, 'printer': printer}).set(1)
```

Comment:
```suggestion
            classes.labels(class=cups_class, printer=printer).set(1)
```
?

Reply:
Unfortunately `class` is a reserved term, so this causes a syntax error. I can add a comment for this.